src/App.tsx

```diff
@@ -56,34 +55,29 @@ function App() {
         },
     }, [client, unlinkContactFn]);
 
-    if (!client || isLoading) {
+    if (!client || isLoading && !context) {
```

Comment:
Why do we need this extra chcek. If it's loading, surely we just show the spinner?

Reply:
You are right, no needed.